web/models/brand.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Askbywho.Brand do
+  use Askbywho.Web, :model
+  alias Askbywho.Email
+
+  schema "brands" do
+    field :name, :string
+    many_to_many :emails, Email, join_through: "brands_emails"
+    timestamps
+  end
+
+  @doc """
+  Creates a changeset based on the `model` and `params`.
+
+  If no params are provided, an invalid changeset is returned
+  with no validation performed.
+  """
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:name])
+    |> validate_required([:name])
```

Comment:
What do we do about mix cased brands?  So what if some typed "Hugo Boss" vs "hugo boss"?  Also, we may want to use something like [jaro](https://hexdocs.pm/elixir/String.html#jaro_distance/2) distance to determine if names are similar (found usually when you're about 0.90, its most likely the same name).  That can come later :)